src/BuildScriptGenerator/Node/NodeBashBuildSnippet.sh.tpl

```diff
@@ -93,23 +93,48 @@ fi
 
 cd "$SOURCE_DIR"
 
-echo
-echo "Running '{{ PackageInstallCommand }}'..."
-echo
-{{ PackageInstallCommand }}
-
 {{ if CustomRunBuildCommand | IsNotBlank }}
+	echo
+	echo "Running '{{ PackageInstallCommand }}'..."
+	echo
+	{{ PackageInstallCommand }}
 	echo
 	{{ CustomRunBuildCommand }}
 	echo
+{{ else if LernaRunBuildCommand | IsNotBlank }}
+	echo
+	echo "Using Lerna version:"
+	lerna --version
+	echo
+	echo
+	echo "Running '{{ LernaInitCommand }} & {{ LernaBootstrapCommand }}':"
+	{{ LernaInitCommand }}
+	{{ LernaBootstrapCommand }}
+	echo
+	echo
+	echo "Running '{{ LernaRunBuildCommand }}'..."
```

Comment:
I found this article and was reading more about lerna ...
https://www.jannikbuschke.de/blog/monorepo-with-lerna-react-and-typescript/

> {
>   "name": "root",
>   "private": true,
>   "devDependencies": {
>     "lerna": "3.13.1"
>   },
>   "scripts": {
>     "bootstrap": "lerna bootstrap --hoist",
>     "start": "lerna run start --stream",
>     "tsc": "lerna run tsc --stream",
>     "watch": "lerna run watch --stream"
>   }
> }
> 
> 

Can we have a test on an app which has a package.json like this to see if this buildscript is good enough for that?

Reply:
try to build/test with this https://github.com/jannikbuschke/lerna-react-typescript-sample